homeassistant/helpers/config_validation.py

```diff
@@ -1101,10 +1101,13 @@ def raise_issue() -> None:
             HOMEASSISTANT_DOMAIN,
             f"integration_key_no_support_{domain}",
             is_fixable=False,
-            severity=IssueSeverity.ERROR,
             issue_domain=domain,
+            severity=IssueSeverity.ERROR,
             translation_key="integration_key_no_support",
-            translation_placeholders={"domain": domain},
+            translation_placeholders={
+                "domain": domain,
+                "add_integration": f"/_my_redirect/config_flow_start?domain={domain}",
```

Comment:
Sorry, we don't have to go through the My handling:

```suggestion
                "add_integration": f"/config/integrations/add?domain={domain}",
```

Reply:
Right, but that makes the link prone to breaking which will be very difficult to track down

Reply:
Using my is also prone to breaking, as that requires you to have that integration enabled.
But should this not be like this:
```suggestion
                "add_integration": f"/config/integrations/dashboard/add?domain={domain}",
```
Ref <https://github.com/home-assistant/frontend/pull/16689/files>

Reply:
@ludeeus that change is precisely why I prefer a my-link. I think we should allow using my-links also when the my integration is disabled for internal links since the perceived privacy issue of my.home-assistant.io does not exist then.